spec/API_specification/linear_algebra_functions.md

```diff
@@ -2,6 +2,12 @@
 
 > Array API specification for linear algebra functions.
 
+```{note}
```

Comment:
Is this note necessary given L11 which states that

> A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.

This already implies that these linear algebra functions are required.